bpf-gpl/icmp.h

```diff
@@ -181,7 +181,7 @@ static CALI_BPF_INLINE int icmp_v4_too_big(struct __sk_buff *skb)
 		__be16  mtu;
 	} frag = {
 		// ICMP MTU ignores the ethernet header.
-		.mtu = host_to_be16(TUNNEL_MTU - sizeof(struct ethhdr)),
+		.mtu = host_to_be16(TUNNEL_MTU),
```

Comment:
I don't understand this change.  The MTU that we signal in the ICMP message needs to be the IP MTU but the TUNNEL_MTU value was meant to hold Felix's config value, which is the ethernet MTU of the tunnel.